Build error on "./hack/build.sh"

**Describe the bug**
I just follow the guide [https://github.com/tkestack/galaxy/blob/master/doc/building.md](url) to build binaries, but on command `./hash/build.sh` getting errors: 

```log
root@d49c9bf924fe:/go/src/tkestack.io/galaxy# ./hack/build.sh
Building galaxy
go build -o bin/galaxy tkestack.io/galaxy/cmd/galaxy
# tkestack.io/galaxy/pkg/utils/iptables
pkg/utils/iptables/iptables_unsupported.go:42:6: grabIptablesLocks redeclared in this block
        previous declaration at pkg/utils/iptables/iptables_linux.go:68:62
pkg/utils/iptables/iptables_unsupported.go:46:6: grabIptablesFileLock redeclared in this block
        previous declaration at pkg/utils/iptables/iptables_linux.go:115:39
```

**Environment**
 - OS: Linux VM_149_11_centos 3.10.107-1-tlinux2_kvm_guest-0049 #1 SMP Tue Jul 30 23:46:29 CST 2019 x86_64 x86_64 x86_64 GNU/Linux
 - golang: go version go1.12.4 linux/amd64

--- a/doc/building.md
+++ b/doc/building.md
@@ -32,3 +32,21 @@ hack/build-rpm.sh
 ```
 BINARY=cmd/galaxy hack/build-single.sh
 ```
+
+# FAQ
+Q: Cannot fetch some modules like 'modernc.org/golex', 'modernc.org/xc', etc?
+A: The `build.sh` script will call `go mod` command to download these modules from gitlab.com,
+   but there are some compatibility problem for `go mod` when resolving gitlab.com addresses, 
+   To fix it, add belowing configuration to `~/.gitconfig` :
+```
+[url "https://gitlab.com/cznic/mathutil.git"]
+        insteadOf = https://gitlab.com/cznic/mathutil
+[url "https://gitlab.com/cznic/xc.git"]
+        insteadOf = https://gitlab.com/cznic/xc
+[url "https://gitlab.com/cznic/strutil.git"]
+        insteadOf = https://gitlab.com/cznic/strutil
+[url "https://gitlab.com/cznic/golex.git"]
+        insteadOf = https://gitlab.com/cznic/golex
+[url "https://gitlab.com/cznic/cc.git"]
+        insteadOf = https://gitlab.com/cznic/cc
+```

--- a/go.mod
+++ b/go.mod
@@ -40,7 +40,7 @@ require (
 	k8s.io/kube-openapi v0.0.0-20190918143330-0270cf2f1c1d // indirect
 	k8s.io/kubernetes v1.16.0-alpha.0
 	k8s.io/utils v0.0.0-20191010214722-8d271d903fe4
-	tkestack.io/tapp-controller v0.0.0-20191017074115-bdf9b4742398
+	tkestack.io/tapp master
 )
 
 replace (

--- a/hack/updatevendor.sh
+++ b/hack/updatevendor.sh
@@ -3,7 +3,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-go env -w GONOSUMDB=git.tencent.com/tke/tapp-controller
+go env -w GONOSUMDB=tkestack.io/tapp
 go mod tidy
 
 #replace (

--- a/pkg/ipam/schedulerplugin/floatingip_plugin_test.go
+++ b/pkg/ipam/schedulerplugin/floatingip_plugin_test.go
@@ -45,8 +45,8 @@ import (
 	. "tkestack.io/galaxy/pkg/ipam/schedulerplugin/testing"
 	"tkestack.io/galaxy/pkg/ipam/schedulerplugin/util"
 	"tkestack.io/galaxy/pkg/utils/database"
-	fakeTAppCli "tkestack.io/tapp-controller/pkg/client/clientset/versioned/fake"
-	tappInformer "tkestack.io/tapp-controller/pkg/client/informers/externalversions"
+	fakeTAppCli "tkestack.io/tapp/pkg/client/clientset/versioned/fake"
+	tappInformer "tkestack.io/tapp/pkg/client/informers/externalversions"
 )
 
 const (

--- a/pkg/ipam/schedulerplugin/resync.go
+++ b/pkg/ipam/schedulerplugin/resync.go
@@ -32,7 +32,7 @@ import (
 	"tkestack.io/galaxy/pkg/ipam/floatingip"
 	"tkestack.io/galaxy/pkg/ipam/schedulerplugin/util"
 	"tkestack.io/galaxy/pkg/utils/database"
-	tappv1 "tkestack.io/tapp-controller/pkg/apis/tappcontroller/v1"
+	tappv1 "tkestack.io/tapp/pkg/apis/tappcontroller/v1"
 )
 
 func (p *FloatingIPPlugin) storeReady() bool {

--- a/pkg/ipam/schedulerplugin/tapp.go
+++ b/pkg/ipam/schedulerplugin/tapp.go
@@ -24,7 +24,7 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	glog "k8s.io/klog"
 	"tkestack.io/galaxy/pkg/ipam/schedulerplugin/util"
-	tappv1 "tkestack.io/tapp-controller/pkg/apis/tappcontroller/v1"
+	tappv1 "tkestack.io/tapp/pkg/apis/tappcontroller/v1"
 )
 
 func tAppFullName(tapp *tappv1.TApp) string {

--- a/pkg/ipam/schedulerplugin/types.go
+++ b/pkg/ipam/schedulerplugin/types.go
@@ -25,8 +25,8 @@ import (
 	list "tkestack.io/galaxy/pkg/ipam/client/listers/galaxy/v1alpha1"
 	"tkestack.io/galaxy/pkg/ipam/floatingip"
 	"tkestack.io/galaxy/pkg/utils/database"
-	"tkestack.io/tapp-controller/pkg/client/clientset/versioned"
-	"tkestack.io/tapp-controller/pkg/client/listers/tappcontroller/v1"
+	"tkestack.io/tapp/pkg/client/clientset/versioned"
+	"tkestack.io/tapp/pkg/client/listers/tappcontroller/v1"
 )
 
 type PluginFactoryArgs struct {

--- a/pkg/ipam/server/server.go
+++ b/pkg/ipam/server/server.go
@@ -51,8 +51,8 @@ import (
 	"tkestack.io/galaxy/pkg/ipam/server/options"
 	"tkestack.io/galaxy/pkg/utils/httputil"
 	pageutil "tkestack.io/galaxy/pkg/utils/page"
-	tappVersioned "tkestack.io/tapp-controller/pkg/client/clientset/versioned"
-	tappInformers "tkestack.io/tapp-controller/pkg/client/informers/externalversions"
+	tappVersioned "tkestack.io/tapp/pkg/client/clientset/versioned"
+	tappInformers "tkestack.io/tapp/pkg/client/informers/externalversions"
 )
 
 type JsonConf struct {

--- a/pkg/utils/iptables/iptables_linux.go
+++ b/pkg/utils/iptables/iptables_linux.go
@@ -1,3 +1,5 @@
+// +build linux
+
 /*
  * Tencent is pleased to support the open source community by making TKEStack available.
  *
@@ -14,7 +16,6 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-// +build linux
 
 /*
 Copyright 2017 The Kubernetes Authors.

--- a/pkg/utils/iptables/iptables_unsupported.go
+++ b/pkg/utils/iptables/iptables_unsupported.go
@@ -1,3 +1,5 @@
+// +build !linux
+
 /*
  * Tencent is pleased to support the open source community by making TKEStack available.
  *
@@ -14,7 +16,6 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-// +build !linux
 
 /*
 Copyright 2017 The Kubernetes Authors.

